README.rst

```diff
@@ -6,7 +6,13 @@ journals  |Travis|_ |Codecov|_
 .. |Codecov| image:: http://codecov.io/github/edx/journals/coverage.svg?branch=master
 .. _Codecov: http://codecov.io/github/edx/journals?branch=master
 
-The ``README.rst`` file should start with a brief description of the repository, which sets it in the context of other repositories under the ``edx`` organization. It should make clear where this fits in to the overall edX codebase. You may also want to provide a brief overview of the code in this repository, including the main components and useful entry points for starting to understand the code in more detail, or link to a comparable description in your repo's docs.
+The Journal service is a standalone IDA that provides functionality to author, market and view Content Journals. This repository contains the needed code to support the base Journal functionality. It uses the Content Mangement System, Wagtail 1.3, to create and author Journals.
```

Comment:
What's Wagtail 1.3? If your listing the version number, it's 1.13, but I don't think we should have the version number in the README.